pkgs/applications/science/electronics/smuview/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libtool
+, pkg-config
+, cmake
+, glib
+, boost
+, libsigrok
+, libserialport
+, libzip
+, libftdi1
+, hidapi
+, glibmm
+, python3
+, qtsvg
+, qwt
+, qttools
```

Comment:
smuview builds for me without libtool, qtsvg, qttools.

Reply:
I looked into the [building guide](https://sigrok.org/wiki/Building#SmuView) for the dependencies, but I guess they might be outdated as well.

I'll remove them.